src/Integration.Vsix/Events/ProjectDocumentsEventsListener.cs

```diff
@@ -134,18 +141,17 @@ public void Initialize()
     {
         return VSConstants.S_OK;
     }
-    
+
     public void Dispose()
     {
-        Dispose(disposing: true);
-        GC.SuppressFinalize(this);
-    }
-    
-    protected virtual void Dispose(bool disposing)
-    {
-        if (disposing)
+        if (isDisposed)
```

Comment:
What's your opinion on `if (disposed)` vs `if (isDisposed)`?

Reply:
I don't have any strong preferences, as long as we are consistent with the rest of the codebase, it should be fine. Here I think is also better the isDisposed because disposed is used as a param of Dispose(bool dispose).

isDisposed is used in ActiveSolutionTracker